TF_Service_dotNet/TouchFreeTests/PositioningModuleTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Numerics;
+using NUnit.Framework;
+using TouchFreeTests.TestImplementations;
+using Ultraleap.TouchFree.Library;
+using Ultraleap.TouchFree.Library.Interactions;
+using Ultraleap.TouchFree.Library.Interactions.PositionTrackers;
+
+namespace TouchFreeTests
+{
+    class PositioningModuleTests
+    {
+        PositioningModule sut;
+        TestVirtualScreenManager virtualScreenManager = new TestVirtualScreenManager();
+
+        public PositioningModuleTests()
+        {
+            // TODO: Remove dependency on the Trackers here and instead use vector3 from mocked tracker instead of hand coordinates to make the tests simpler
```

Comment:
This is a bit of a nice to have, I can put this in as part of this PR, but as it is the two tests for calculate positions will also test the index tip position tracker, I originally used that as it had the most simple hand to position conversion. I could create a test instance and directly return the position without exercising any tracker code instead (what this TODO is referencing)

Reply:
I have done this